package-lock.json

```diff
@@ -10129,13 +10129,13 @@
       "dependencies": {
```

Comment:
As per https://docs.opencollective.com/help/developers/guidelines, `package-lock.json` shouldn't be commit. Best way to un-commit it is to rebase you pull request, let me know if you want me to handle that.